### PR TITLE
Add check whether images are located in the doc/ folder

### PR DESCRIPTION
**Pull request by fwolter**
Signed-off-by: Fabian Wolter <github@fabian-wolter.de>

### CODE REVIEW

File: custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/readme/MarkdownVisitor.java [hunk 1]
```diff
@@ -156,6 +158,13 @@ public void visit(ListBlock list) {
         processListBlock(list);
     }
 
+    public void visit(Image image) {
+        String url = image.getUrl().toString();
+        if (!url.startsWith("doc/") && !url.startsWith("http://")) {
```
Comment (wborn) [inline custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/readme/MarkdownVisitor.java:163]:
```suggestion
        if (!url.startsWith("doc/") && !url.startsWith("http://") && !url.startsWith("https://")) {
```

**Review by wborn — COMMENTED** [review]
Looks like a handy check to me. :+1:

File: custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/readme/MarkdownVisitor.java (hunk 1 above)
Comment (fwolter) [reply custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/readme/MarkdownVisitor.java]: Sure, fixed.

**Review by wborn — APPROVED** [review]
Thanks for the new check!

**Pull request merged**